changelog.d/947.removal

```diff
@@ -0,0 +1 @@
+Removed `upgrade-db-0.*.js` scripts. Use an older release to upgrade NeDB.
```

Comment:
This is not very helpful - can put a link to where people can learn about the upgrade process please?